django/db/models/query.py

```diff
@@ -1137,6 +1130,21 @@ def _has_filters(self):
         """
         return self.query.has_filters()
 
+    def _check_allowed(self, action):
```

Comment:
It might be worth making the name a bit more descriptive such as `check_queryset_method_allowed`.